lib/rspec/core/configuration.rb

```diff
@@ -473,8 +473,7 @@ def initialize
       # Used to set higher priority option values from the command line.
       def force(hash)
         ordering_manager.force(hash)
-
-        @preferred_options.merge!(hash).merge!(prepared_streams(hash))
+        @preferred_options.merge!(hash)
 
         return unless hash.key?(:example_status_persistence_file_path)
         clear_values_derived_from_example_status_persistence_file_path
```

Comment:
WDYT of changing `reset` so that it delegates to `reset_reporter` to reset the 3 ivars that method takes care of, instead of manually resetting those same 3 ivars?